.github/workflows/tests-smoke.yaml

```
@@ -68,6 +68,11 @@ jobs:
         with:
           persist-credentials: false
 
+      - name: Helm Chart Smoke Test Common label
+        run: |
+          echo "Checking Helm templates for common label"
+          make -C install/kubernetes lint
+          
       - name: Run helm-charts
         run: |
           make -C install/kubernetes
```

```suggestion
          
      - name: Run helm-charts
        run: |
          make -C install/kubernetes
```
No need for the changes in this file - running `make -C install kubernetes` runs the `all` target there which includes the `lint` target.